src/services/ldap/strategies/iserv-idm.js

```diff
@@ -30,8 +30,8 @@ class IservIdmLDAPStrategy extends AbstractLDAPStrategy {
 	 * (Array) roles = ['teacher', 'student', 'administrator']
 	 * @memberof GeneralLDAPStrategy
 	 */
-	verifyConfiguration(school) {
-		return this.getUsers(school);
+	verifyConfiguration() {
+		return this.getUsers();
```

Comment:
I am a bit confused that the method itself has not been adjusted. Doesn't that lead to an error if the school argument is used there?